components/Tabs/Tabs.js

```diff
@@ -1,72 +1,56 @@
 class TabLink {
   constructor(tabElement){
-    // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
-    // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
-    
-    // We need to find out if a user clicked 'all' cards or a specific category.  Follow the instructions below to accomplish this task:    
-    
-    /* <- Delete this comment block when you work on the if statement
-    // Check to see if this.tabData is equal to 'all'
-    if(){
-      // If `all` is true, select all cards regardless of their data attribute values
-      // this.cards = ;
-    } else {
-      // else if `all` is false, only select the cards with matching this.tabData values
-      // this.cards = ;
+      this.tabElement = tabElement;
+      this.tabData = tabElement.dataset.tab;
+      
+      if (this.tabData === 'all') {
+          this.cards = document.querySelectorAll('.card');
+      } else {
+          this.cards = document.querySelectorAll(`.card[data-tab='${this.tabData}']`);
+      }
+      
+      this.cards.forEach( card =>
```

Comment:
Be sure to read the instructions. You should be mapping over the NodeList to convert the cards into TabCards:

`// Map over the newly converted NodeList we just created in our if statement above. Convert each this.cards element into a new instance of the TabCard class. Pass in a card object to the TabCard class.`